h/httpserver.h

```diff
@@ -212,6 +212,7 @@ typedef struct HttpServer_tag{
   char             *defaultProductURLPrefix;
   uint64           serverInstanceUID;   /* may be something smart at some point. Now just startup STCK */
   void             *sharedServiceMem; /* address shared by all HttpServices */
+  hashtable        *loggingTable;
```

Comment:
That should really be pluginLoggingTable. Variable names should be self documenting as much as possible.